test/foundry/FleekERC721/FleekERC721.ap.t.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import "./TestBase.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {FleekAccessControl} from "contracts/FleekAccessControl.sol";
+
+contract Test_FleekERC721_AccessPoint is Test_FleekERC721_Base {
+    using Strings for address;
+    uint256 internal tokenId;
+
+    function assertAccessPointJSON(
+        string memory accessPointName,
+        string memory _tokenId,
+        string memory score,
+        string memory nameVerified,
+        string memory contentVerified,
+        address owner
+    ) internal {
+        string memory current = CuT.getAccessPointJSON(accessPointName);
+        // prettier-ignore
+        string memory expectedJSON = string(abi.encodePacked('{"tokenId":', _tokenId, ',"score":', score, ',"nameVerified":', nameVerified, ',"contentVerified":', contentVerified, ',"owner":"', owner.toHexString(), '"}'));
+        assertEq(current, expectedJSON);
+    }
+
+    function setUp() public {
+        baseSetUp();
+        tokenId = mintDefault(deployer);
+    }
+
+    function test_addAccessPoint() public {
+        CuT.addAccessPoint(tokenId, "accesspoint.com");
+    }
+
+    function test_getAccessPointJSON() public {
+        string memory accessPointName = "accesspoint.com";
+        CuT.addAccessPoint(tokenId, accessPointName);
+
+        assertAccessPointJSON(accessPointName, "0", "0", "false", "false", deployer);
```

Comment:
This is what I meant by my comment earlier about checking the existence of the access point. Maybe `test_addAccessPoint` and `test_getAccessPointJSON` can be merged into one test case since they share a similar purpose and the latter complements the former.

Reply:
I will be removing the `test_addAccessPoint`